code/modules/mob/mob_helpers.dm

```diff
@@ -1,114 +1,17 @@
-
-// fun if you want to typecast humans/monkeys/etc without writing long path-filled lines.
-/proc/ishuman(A)
-	if(istype(A, /mob/living/carbon/human))
-		return 1
-	return 0
-
 /proc/issmall(A)

```

Comment:
you could technically make this into a define with something like
`#define issmall(A) (A:species:is_small)`